packages/labs/ssr-react/src/lib/node/wrap-create-element.ts

```diff
@@ -40,11 +42,20 @@ export function wrapCreateElement(
           },
         });
 
+        const newChildren: ReactNode[] = [templateShadowRoot];
+        if (props?.children !== undefined) {
+          if (Array.isArray(props.children)) {
+            newChildren.push(...props.children);
+          } else {
+            newChildren.push(props.children);
+          }
+        }
+        newChildren.push(...children);
```

Comment:
Did you say that the native behavior was do override props.children with the children argument if it was present? Should we emulate that?

Reply:
My assumption was that tooling that generates `React.createElement()` calls should always do one or the other so it felt harmless for our code to "combine" them but I suppose emulating React makes sense.